hobbymatcher/src/main/java/com/hobbymatcher/controller/user/UserController.java

```diff
@@ -33,27 +33,33 @@ public UserController(UserService userService) {
         List<User> list = new ArrayList<User>();
         try {
             list = userService.getUserList();
-            modelMap.put("rows", list);
-            modelMap.put("total", list.size());
+            modelMap.put("list", list);
+            modelMap.put("code", 200);
         } catch (Exception e) {
             e.printStackTrace();
             modelMap.put("success", false);
-            modelMap.put("errMsg", e.toString());
+            modelMap.put("code", "400");
```

Comment:
I think we should use something like this to indicate the status code in the response headers - https://stackoverflow.com/questions/16232833/how-to-respond-with-http-400-error-in-a-spring-mvc-responsebody-method-returnin